src/Asset/Widget/Queue.php

```diff
@@ -99,7 +99,7 @@ public function process(Request $request, Response $response)
     {
         /** @var WidgetAssetInterface $widget */
         foreach ($this->queue as $widget) {
-            if ($widget->getZone() === Zone::FRONTEND && $widget->isDeferred()) {
+            if ($widget->isDeferred()) {
                 $this->addDeferredJavaScript($widget, $response);
```

Comment:
I _guess_ because during the refactor they didn't actually work as deferred in the backend. I did some monkey-testing, and at the moment it seems to work fine in my tests.